blog/20210808-mate-1-26-released.md

```diff
@@ -10,7 +10,7 @@
 
 This release might have taken a little longer than usual but now after 18 month of development we are very pleased to release MATE 1.26. A big thank you to all contributors who helped to make this happen.
 
-![MATE About dialog](/assets/img/blog/about-mate-1.26.png)
+![MATE About dialog](/assets/img/blog/about-mate-1.26.ja.png)
 
```

Comment:
Main blog/20210808-mate-1-26-released.md should not use Japanese screenshots.

Reply:
I am not suggesting to use about-mate-1.26.ja.png on main English page. But I mean using it on Japanese page, just like Catala, Italian or Spanish pages of "MATE 1.24 release" An example: https://mate-desktop.org/ca/blog/2020-02-10-mate-1-24-released/
What do you think?

Reply:
Yes, good idea, but i think this change should be done at 20210808-mate-1-26-released.ja.md or not ?

Reply:
See Catalan 1.24 release page https://raw.githubusercontent.com/mate-desktop/mate-desktop.org/master/blog/20200210-mate-1-24-released.ca.md  , for example

Reply:
Change at Transifex is done. I thought it would be better to fix 20210808-mate-1-26-released.ja.md after about-mate-1.26.ja.png was added to the master of mate-desktop.org/files/assets/img/blog/.
Thank you.